build/nuke/Extensions/TargetFrameworksExtensions.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Extensions
+{
+    public static class TargetFrameworksExtensions
+    {
+        public static IEnumerable<TargetFramework> FilterWindowsOnly(this IEnumerable<TargetFramework> frameworks)
+        {
+            return frameworks.Except(TargetFramework.WindowsOnly);
+        }
+
+        public static IEnumerable<TargetFramework> FilterNetStandard(this IEnumerable<TargetFramework> frameworks)
+        {
+            return frameworks.Except(TargetFramework.NetStandard);
+        }
+
+        public static IEnumerable<TargetFramework> CrossPlatformTestable(this IEnumerable<TargetFramework> frameworks)
+        {
+            return frameworks
+                .FilterWindowsOnly()
+                .FilterNetStandard();
```

Comment:
Why are we excluding netstandard 2.0 from the corss platform tests? Netstandard 2.0 libraries should work across multiple platforms.

Reply:
I think that `netstandard2.0` is not needed at all. See: https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/232